src/accounts/AccountPage.tsx

```diff
@@ -170,9 +154,6 @@ const AccountAboutPage: FC = () => {
             </FlexBox>
           </>
         )}
-        <Overlay visible={isSwitchAccountVisible}>
-          <SwitchAccountOverlay onDismissOverlay={closeSwitchAccountDialog} />
-        </Overlay>
         <CancelServiceProvider>
```

Comment:
We don't need the overlay or `closeSwitchAccountDialog` either, because there is no longer a switch-account button on this page.